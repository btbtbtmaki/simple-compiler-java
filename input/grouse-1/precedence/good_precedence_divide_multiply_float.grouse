main {

  print 6.2 / 3.9 * 4.7 nl;
  print 6.6 / 4.6 * 4.4 nl;
  print 7.5 / 2.3 * 9.3 nl;
  print 5.6 / 9.6 * 1.9 nl;
  print 7.1 / 5.2 * 8.9 nl;
  print 2.9 / 7.9 * 5.8 nl;
  print 4.0 / 3.6 * 8.2 nl;
  print 2.3 / 7.2 * 8.1 nl;
  print 7.7 / 5.4 * 2.6 nl;
  print 4.8 / 5.3 * 8.5 nl;
  print 6.3 / 4.0 * 3.9 nl;
  print 9.5 / 1.6 * 8.6 nl;

  print 7.3 * 1.1 / 8.9 nl;
  print 4.8 * 8.7 / 2.5 nl;
  print 9.9 * 4.6 / 6.9 nl;
  print 1.8 * 8.1 / 9.6 nl;
  print 2.8 * 9.4 / 4.8 nl;
  print 3.2 * 3.5 / 9.9 nl;
  print 2.9 * 5.3 / 8.3 nl;
  print 3.8 * 6.8 / 9.4 nl;
  print 7.7 * 2.0 / 8.5 nl;
  print 7.1 * 5.5 / 2.2 nl;
  print 5.5 * 1.6 / 6.4 nl;
  print 5.8 * 6.1 / 2.3 nl;

  print 9.6 / 7.0 * 2.4 / 7.5 nl;
  print 9.2 / 8.9 * 8.8 / 7.2 nl;
  print 3.0 / 9.5 * 5.1 / 5.2 nl;
  print 2.2 / 2.9 * 2.2 / 8.6 nl;
  print 7.4 / 7.6 * 4.5 / 6.8 nl;
  print 7.5 / 8.6 * 6.5 / 7.8 nl;
  print 5.5 / 7.1 * 7.3 / 8.8 nl;
  print 4.5 / 4.5 * 1.1 / 3.3 nl;
  print 3.5 / 7.0 * 6.2 / 3.9 nl;
  print 3.8 / 9.8 * 4.7 / 8.3 nl;
  print 7.2 / 9.8 * 1.4 / 4.7 nl;
  print 1.4 / 4.0 * 6.6 / 2.6 nl;

  print 3.4 * 5.5 / 8.0 * 9.3 nl;
  print 9.9 * 9.9 / 9.1 * 1.7 nl;
  print 7.2 * 6.2 / 1.8 * 1.7 nl;
  print 1.5 * 4.9 / 5.1 * 3.7 nl;
  print 6.3 * 5.2 / 4.0 * 1.0 nl;
  print 9.0 * 6.8 / 8.9 * 4.0 nl;
  print 6.0 * 8.4 / 6.0 * 4.8 nl;
  print 8.8 * 1.7 / 4.5 * 2.9 nl;
  print 2.2 * 5.4 / 1.5 * 5.8 nl;
  print 3.5 * 8.5 / 3.6 * 1.4 nl;
  print 5.5 * 1.0 / 8.3 * 2.2 nl;
  print 8.8 * 4.3 / 8.9 * 2.8 nl;

}

