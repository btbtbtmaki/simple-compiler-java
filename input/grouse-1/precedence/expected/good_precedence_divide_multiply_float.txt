7.47179
6.31304
30.3261
1.10833
12.1519
2.12911
9.11111
2.5875
3.70741
7.69811
6.1425
51.0625
0.902247
16.704
6.6
1.51875
5.48333
1.13131
1.85181
2.74894
1.81176
17.75
1.375
15.3826
0.438857
1.26342
0.309717
0.194066
0.64435
0.726744
0.642606
0.333333
0.794872
0.219572
0.218845
0.888462
21.7388
18.3096
42.16
5.33235
8.19
27.5056
40.32
9.64089
45.936
11.5694
1.45783
11.9047
